packages/ui/src/components/AccountEditName.tsx

```diff
@@ -66,6 +67,7 @@ const AccountEditName = ({ address, onNameChange, className }: Props) => {
           onChange={onChange}
           disabled={isExtensionAccount}
           value={name}
+          data-cy={`input-edit-${testId}-name`}
```

Comment:
even though passing the testId as a prop I'm still using data-cy as it's an internal component, I believe that's what we said to do in the past (Only use data-testid when it's an external component).